share/tools/web_config/sample_prompts/robbyrussell.fish

```diff
@@ -6,7 +6,12 @@ function fish_prompt
     if not set -q -g __fish_robbyrussell_functions_defined
         set -g __fish_robbyrussell_functions_defined
         function _git_branch_name
-            echo (git symbolic-ref HEAD ^/dev/null | sed -e 's|^refs/heads/||')
+            set -l branch (git symbolic-ref --quiet HEAD ^/dev/null)
+            if test $branch
+                echo $branch | sed -e 's|^refs/heads/||'
```

Comment:
Since you're touching this please change it to use the `string` builtin so we're not executing an external command where that can be avoided.